test-app/src/main/java/com/mapbox/navigation/examples/core/MapboxSnapshotActivity.java

```diff
@@ -305,10 +250,6 @@ private void updateLocation(List<Location> locations) {
     );
   }
 
-  private LocationPluginImpl getLocationComponent() {
-    return mapView.getPlugin(LocationPluginImpl.class);
-  }
-
   private CameraAnimationsPlugin getMapCamera() {
     return CameraAnimationsPluginImplKt.getCameraAnimationsPlugin(mapView);
```

Comment:
Maps should add `@file:JvmName("CameraAnimationsPluginImpl")` to make it more Java user-friendly (remove the need of `Kt`) cc @tobrun